config/redirects

```diff
@@ -22,3 +22,5 @@ raw: ${prefix}/stable -> ${base}/current/
 
 [*-v5.5]: ${prefix}/${version}/fundamentals/run-command/ -> ${base}/${version}/usage-examples/command/
 [*-master]: ${prefix}/${version}/fundamentals/crud/write-operations/change-a-document/ -> ${base}/${version}/fundamentals/crud/write-operations/modify/
+
+[*-v5.8]: ${prefix}/${version}/fundamentals/connection/socks/ -> ${base}/v6.0/fundamentals/connection/socks/
```

Comment:
Issue:

I think this should not switch to the v6.0 documentation when browsing an older version of the docs since it can cause confusion.

Suggestion:

I think this should link to the landing page because the user is attempting to access a page that doesn't exist for a specific version.
Also, I think that there might be v5.x minor releases that come after v5.8, so it should apply to any version up to the first version in which it was implemented (v6.0). Note that the round parens is "exclusive" in the definition:

```
[*-v6.0): ${prefix}/${version}/fundamentals/connection/socks/ -> ${base}/${version}
```